CMakeLists.txt

```diff
@@ -149,6 +149,23 @@ install(FILES "${CMAKE_SOURCE_DIR}/Contributors.md"
 install(FILES "docs/scap-security-guide.8"
     DESTINATION "${CMAKE_INSTALL_MANDIR}/man8")
 
+if (UNIX)
+    install(CODE "
+        file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/xml/scap/ssg/)
+    ")
+    install(CODE "
+        execute_process(COMMAND ln -sf ../../../scap/ssg ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/xml/scap/ssg/content)
+    ")
+endif()
+
+add_custom_target(clean-all
+    COMMAND make -C ${CMAKE_BINARY_DIR}/ clean
+    COMMAND rm -f ${CMAKE_BINARY_DIR}/*.html
+    COMMAND rm -f ${CMAKE_BINARY_DIR}/*.xml
+    COMMAND find ${CMAKE_BINARY_DIR}/ -type d -name oval | xargs rm -rf
+    COMMAND find ${CMAKE_BINARY_DIR}/ -type d -name remediations | xargs rm -rf
+)
```

Comment:
That's a bug in the cmake targets then, we have side-effects not listed in cmake. Let's not do workarounds to hide bugs pls.